API_Contracts.md

```diff
@@ -72,6 +74,28 @@ Code 200
 
 ---
 
+Used for Login
+
+**`POST /api/v1/users/tokens`**
```

Comment:
Is such endpoint a good choice? Maybe it's better to simplify it as /api/v1/login

Reply:
Yes, we could simplify it